Sources/Workspace/WindowsToolchainInfo.swift

```diff
@@ -74,6 +74,10 @@ public struct WindowsPlatformInfo {
         /// XCTEST_VERSION
         /// specifies the version string of the bundled XCTest.
         public let xctestVersion: String
+
+        /// SWIFTC_FLAGS
+        /// Specifies extra flags to pass to swiftc from Swift Package Manager.
```

Comment:
Is it worth specifying that these flags get added at the end of the command line?  (so any search paths that are added here would end up after the others, etc)

Reply:
I think for the time being, I would really rather leave it underspecified.  I don't want anyone to actually start depending on the behaviour of these items just yet.  Once we get more stable, I think that extending the documentation and solidifying the behaviour makes sense.